src/components/Menus/Menu.js

```diff
@@ -31,37 +31,50 @@ class Menu extends React.Component {
     /** Role HTML attribute */
     role: PropTypes.string,
     /** Customize style of menu parent */
-    style: PropTypes.shape({})
+    style: PropTypes.shape({}),
   }
 
   static defaultProps = {
     role: 'menu',
     ariaHidden: false,
     style: {},
-    onSelect: () => {} // eslint-disable-line no-empty-function
+    onSelect: () => {}, // eslint-disable-line no-empty-function
   }
 
   state = {
-    currentTabIndex: null
+    currentTabIndex: null,
   }
 
-  handleBlur = (event) => {
+  getMenuItemChildren() {
```

Comment:
manual change: `sort-comp` rule wants `set/get` then `on`/`handle`  after the lifecycle methods